configuration/configuration.go

```diff
@@ -613,6 +646,9 @@ vwIDAQAB
 	defaultKeycloakClientID = "fabric8-online-platform"
 	defaultKeycloakSecret   = "7a3d5a00-7f80-40cf-8781-b5b6f2dfd1bd"
 
+	defaultGithubClientID = "7498db1eaea3b051fbba"
```

Comment:
I created a test github client in my account. We should eventually use a github client details from `fabric8-services`